src/containers/WelcomePage/components/worklist/SubjectDropdown.tsx

```diff
@@ -42,13 +48,20 @@ const SubjectDropdown = ({ subjectIds, filterSubject, setFilterSubject }: Props)
   );
   const subjectContexts = useMemo(() => {
     if (subjects?.results.length) {
-      const archivedAtBottom = sortBy(
-        subjects.results,
-        (r) => r.metadata.customFields.subjectCategory === 'archive',
-      );
-      return archivedAtBottom.map((r) => ({ value: r.id, label: r.name }));
+      let updatedArchived;
+      if (removeArchived) {
+        updatedArchived = subjects.results.filter(
+          (s) => s.metadata.customFields.subjectCategory !== 'archive',
+        );
+      } else {
+        updatedArchived = sortBy(
+          subjects.results,
+          (r) => r.metadata.customFields.subjectCategory === 'archive',
+        );
+      }
```

Comment:
Er det meininga at denne skal enten vise alt som ikkje er arkivert eller alt som er arkivert?

Reply:
Eller er det slik at arkiverte sorteres sist?

Reply:
Sånn det har vært tidligere så har arkiverte blitt sortert sist, ønsket var at i favoritter-blokken så skal utgåtte fag skjules helt, så legger til et option for å filtrere ut arkiverte og samtidig beholde oppførselen om at de sorteres nederst ellers. Virket som det må tas en runde på om det kanskje skal fjernes overalt og etterhvert ..